app/views/ChooseProvider.js

```diff
@@ -1,3 +1,4 @@
+import _ from 'lodash';
```

Comment:
Use the function import instead of the whole library:

`import findIndex from 'lodash/findIndex';`